sdk/core/core/inc/az_http_response_parser.h

```diff
@@ -4,10 +4,10 @@
 #ifndef _az_HTTP_RESPONSE_PARSER_H
 #define _az_HTTP_RESPONSE_PARSER_H
 
+#include <az_span_reader.h>
```

Comment:
run Ctrl+K,D in VS - it will sort these headers. Please run on all files you're changing, so the next time someone else does it, the unneccessary lines do not change.